lib/Driver/ToolChains.cpp

```diff
@@ -1571,16 +1567,10 @@ toolchains::Windows::constructInvocation(const LinkJobAction &job,
     Arguments.push_back(context.Args.MakeArgString(Target));
   }
 
-  bool staticExecutable = false;
-  bool staticStdlib = false;
-
-  if (context.Args.hasFlag(options::OPT_static_executable,
-                           options::OPT_no_static_executable, false)) {
-    staticExecutable = true;
-  } else if (context.Args.hasFlag(options::OPT_static_stdlib,
-                                  options::OPT_no_static_stdlib, false)) {
-    staticStdlib = true;
-  }
+  bool staticExecutable = context.Args.hasFlag(
+      options::OPT_static_executable, options::OPT_no_static_executable, false);
+  bool staticStdlib = context.Args.hasFlag(
+      options::OPT_static_stdlib, options::OPT_no_static_stdlib, false);
```

Comment:
This isn't exactly equivalent: previously `staticExecutable` implied `!staticStdlib`. But that doesn't actually make sense to me; shouldn't it be the other way around?